app/modules/api/actionTypes.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+import type { Error } from 'types/error';
+
+import type { StatusType } from './model';
+
+/* Action constants */
+
+// Reducer actions
+export const SET_STATUS_IN_STATE: 'api/SET_STATUS_IN_STATE_IN_STATE' = 'api/SET_STATUS_IN_STATE_IN_STATE';
```

Comment:
SET_STATUS_IN_STATE**_IN_STATE** is probably a replace error?

Reply:
Oops! Y u approve tho